contracts/strategies/HurricaneStratForLP.sol

```diff
@@ -0,0 +1,265 @@
+    }
+    require(depositToken.transferFrom(msg.sender, address(this), amount));
+    _stakeDepositTokens(amount);
+    _mint(account, getSharesForDepositTokens(amount));
+    totalDeposits = totalDeposits.add(amount);
+    emit Deposit(account, amount);
+  }
+
+  function _safeTransfer(address token, address to, uint256 value) private {
+    require(IERC20(token).transfer(to, value), "HurricaneStrategyForLP::withdraw");
+  }
+  function withdraw(uint amount) external override {
+    uint depositTokenAmount = getDepositTokensForShares(amount);
+    if (depositTokenAmount > 0) {
+      _withdrawDepositTokens(depositTokenAmount);
+      _safeTransfer(address(depositToken), msg.sender, depositTokenAmount);
+      _burn(msg.sender, amount);
+      totalDeposits = totalDeposits.sub(depositTokenAmount);
+      emit Withdraw(msg.sender, depositTokenAmount);
+    }
+  }
+
+  function _withdrawDepositTokens(uint amount) private {
+    require(amount > 0, "HurricaneStrategyForLP::_withdrawDepositTokens");
+    stakingContract.withdraw(PID, amount);
+  }
+
+  function reinvest() external override onlyEOA {
+    uint unclaimedRewards = checkReward();
+    require(unclaimedRewards >= MIN_TOKENS_TO_REINVEST, "HurricaneStrategyForLP::reinvest");
+    _reinvest(unclaimedRewards);
+  }
+
+  /**
+    * @notice Reinvest rewards from staking contract to deposit tokens
+    * @dev Reverts if the expected amount of tokens are not returned from `stakingContract`
+    * @param amount deposit tokens to reinvest
+    */
+  function _reinvest(uint amount) private {
+    stakingContract.deposit(PID, 0);
+    uint devFee = amount.mul(DEV_FEE_BIPS).div(BIPS_DIVISOR);
+    if (devFee > 0) {
+      require(rewardToken.transfer(devAddr, devFee), "HurricaneStrategyForLP::_reinvest, dev");
+    }
+
+    uint adminFee = amount.mul(ADMIN_FEE_BIPS).div(BIPS_DIVISOR);
+    if (adminFee > 0) {
+      require(rewardToken.transfer(owner(), adminFee), "HurricaneStrategyForLP::_reinvest, admin");
+    }
+
+    uint reinvestFee = amount.mul(REINVEST_REWARD_BIPS).div(BIPS_DIVISOR);
+    if (reinvestFee > 0) {
+      require(rewardToken.transfer(msg.sender, reinvestFee), "HurricaneStrategyForLP::_reinvest, reward");
+    }
+
+    uint depositTokenAmount = _convertRewardTokensToDepositTokens(
+      amount.sub(devFee).sub(adminFee).sub(reinvestFee)
+    );
+
+    _stakeDepositTokens(depositTokenAmount);
+    totalDeposits = totalDeposits.add(depositTokenAmount);
+
+    emit Reinvest(totalDeposits, totalSupply);
+  }
+    
+
+  function _stakeDepositTokens(uint amount) private {
+    require(amount > 0, "HurricaneStrategyForLP::amount=0");
+    stakingContract.deposit(PID, amount);
+  }
+
+  function checkReward() public override view returns (uint) {
+    (uint256 pendingAmounts,) = stakingContract.pending(PID, address(this));
+    return pendingAmounts;
+  }
+
+    function getFinalAmountOut(address factory, uint amountIn, address[] memory path) internal view returns (uint amount) {
```

Comment:
some tab issues here, formatting, could you run prettier on the files so we get it consistent?

```
npx prettier --write "contracts/strategies/HurricateStratForLP.sol"
```

If you could, please also format the other files in this PR